Technical-Steering-Committee/release.md

```diff
@@ -1,3 +1,40 @@
+# Zowe Release Schedule
+
+We release Zowe on a regular schedule. Major versions every two years. Minor versions on a schedule published in [Community repository](https://github.com/zowe/community/blob/master/Project%20Management/Schedule/Zowe%20PI%20%26%20Sprint%20Cadence.md). The minor changes shouldn't introduce breaking changes. We introduce the breaking changes only on the border of the major releases. 
+
+As a Zowe community we have three types of releases:
+
+- Maintenance - This release receives only security patches and fixes for the critical bugs as outlined below. It's suitable for environment that prefers stability to new features.  
+- Active - This release receives updates regularly according to the schedule, and it is for the users that are willing to invest a bit more time into upgrades to get the latest features. 
+- Nightly - The Zowe publishes bleeding edge state of the project on the nightly basis. There are no guarantees around the stability of the build. 
+
+The release will be active for 2 years. We will support release in the maintenance mode for 2.5 years. This means that at one point in time we will support one active release and up to two maintenance releases.   
+
+We will provide upgrade path from maintenance release to the maintenance release so that the users will be able to remain only on the maintenance releases. 
+
+![Schedule for the nearest releases](major_releases.png)
+
+## Maintenance release
+
+It is difficult to pre-determine all the conditions that will determine when a fix will be created for a Maintenance release, so this is not to be used as an exhaustive list. Characteristics of what is considered a “critical” bug or defect include:
```

Comment:
The maintenance mode means that we can't introduce the breaking change even more than for the active release.